docs/open-api/credentials.yaml

```diff
@@ -68,7 +59,7 @@ components:
           type: string
           description: The type of the credential, for example, s3-token, s3-secret-key, oss-token, oss-secret-key, gcs-token, adls-token, azure-account-key, etc.
         expireTimeInMs:
-          type: long
+          type: integer
           description: The expiration time of the credential in milliseconds since the epoch, 0 means not expire.
```

Comment:
Gocha. :)

This is not supposed to be a  time elapsed since epoch.
It is supposed to be time duration in milliseconds when the credential (e.g. a certificate) will expire.

Reply:
It's not an time duration, because the credential may need some time to transfer from server side to client side,  which makes duration not correct.